src/index.html

```diff
@@ -60,6 +59,8 @@
         <script src="bower_components/angular-moment/angular-moment.js"></script>
         <script src="bower_components/angular-upload/angular-upload.min.js"></script>
         <script src="assets/js/modernizr-output.js"></script>
+        <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.8.0/highlight.min.js"></script>
```

Comment:
Try to avoid adding libraries from external sources. I'd like to later compile all javascript in one single file (through a grunt task). This can be refactored later though...